### DISCUSSION TITLE
TabBar example in tutorial

### DISCUSSION THREAD

**Discussion by AUSoftware** [unoplatform/uno#14609] [Q&A] [answered]
I read the "How-To: Use a TabBar to Switch Views" at https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Tutorials/Navigation/Advanced/HowTo-UseTabBar.html
It mentions SignUpPage, SignUpViewModel but the code in App.cs is Login page (no SignUp page). Is the code correct?
![Screenshot 2023-11-29 101444](https://github.com/unoplatform/uno/assets/109187841/c395a33a-152c-4000-a30e-151eda45ba1d)

**Comment by nickrandolph** [comment]
@AUSoftware it's not particularly clear but the intent is that you can provide your own SignUpPage. We'll look at what we can to do improve the documentation here. For now, you can add a new page, SignUpPage.xaml, to your application to complete the example.

**Reply by AUSoftware** [reply, marked as answer]
Thanks, I can make it works with the change in App.cs:
```
views.Register(
    new ViewMap(ViewModel: typeof(ShellViewModel)),
    new ViewMap<SignUpPage, SignUpViewModel>(),
    new ViewMap<MainPage, MainViewModel>()
);

routes.Register(
    new RouteMap("", View: views.FindByViewModel<ShellViewModel>(),
        Nested: new RouteMap[]
        {
        new RouteMap("Main", View: views.FindByViewModel<MainViewModel>(),
            Nested: new RouteMap[]
            {
                new RouteMap("signup", View: views.FindByViewModel<SignUpViewModel>())
            })
        }));
```
path name must be lower-case letter "signup". If we set "SignUp" like the "Login", it won't work
